File: schoolBook/Welcome.java
package schoolBook;

public class Welcome {
    public static void run() {
        System.out.println("Welcome to the school magazine");
        while (true) {
            try {
                System.out.println(
                        "\nSelect a user\nDirector enter 1\nTeacher - enter 2\nStudent - enter 3\nExit - enter 4\n");
                int menu = Integer.parseInt(System.console().readLine());
                if (menu == 1) {
                    System.out.println("Director");
                } else if (menu == 2) {
                    System.out.println("Teacher");
                } else if (menu == 3) {
                    Student.run();
                } else if (menu == 4) {
                    System.out.println("Out");
                    break;
                }
            } catch (Exception e) {
                System.out.println("\nYou enter wrong number. Try again!\n");
            }

        }
    }

}

File: Школа/welcome.py
import do_teacher as d_t
import do_director as d_d
import do_student as d_s


def welcome():
    print("Welcome to the school magazine")
    while True:
        try: 
            menu=int(input("Select a user\n\
            Director enter 1\n\
            Teacher - enter 2\n\
            Student - enter 3\n\
            Exit - enter 4\n"))
            if menu==1:
                d_d.start_director()               
            elif menu==2:
                d_t.start_teacher()
            elif menu==3:
                d_s.student()
            elif menu==4:
                break                         
        except ValueError:
            print("\nYou enter wrong number. Try again!") 

   
            